Makefile

```
@@ -24,11 +24,17 @@ fmt-check:
 ci: fmt-check test
 
 OPA_VER=v0.39.0
-OPA_URL=https://openpolicyagent.org/downloads/$(OPA_VER)/opa_linux_amd64_static
+OPA_FILE=opa_linux_amd64_static
+OPA_URL=https://openpolicyagent.org/downloads/$(OPA_VER)/$(OPA_FILE)
+OPA_SHA_FILE=$(OPA_FILE).sha256
+OPA_SHA_URL=$(OPA_URL).sha256
 OPA_DEST=/usr/bin/opa
 
 install-opa:
-	sudo curl -s -L -o $(OPA_DEST) $(OPA_URL)
+	curl -s -L -O $(OPA_URL)
+	curl -s -L -O $(OPA_SHA_URL)
+	sha256sum -c $(OPA_SHA_FILE)
```

Something like this could work:
```suggestion
	echo "73ab60fab6c28131b4fa8f2e7e44973165c2be0c46e3e6feabce68d7eec9335a  opa_linux_amd64_static" | sha256sum --check
```

---

Thanks! Included in new revision.